### PR TITLE
[frigate] chart based on common

### PR DESCRIPTION
**Pull request by onedr0p**
**Description of the change**

Adding Frigate back into our Charts Library, starting at version 5.0.0 due to last Frigate chart was 4.x

**Benefits**

While there is an [official Chart](https://github.com/blakeblackshear/blakeshome-charts) for Frigate, I feel we can do a better job supporting it here using the common library. We will be responsive to changes as the app changes overtime.

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

File: charts/stable/frigate/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.names.fullname" . }}-config
+  labels:
+    {{- include "common.labels" . | nindent 4 }}
+data:
+  config.yml: |
+    {{- .Values.config | nindent 4 }}
```
Comment (bjw-s) [inline charts/stable/frigate/templates/configmap.yaml:9]: Could be nice to allow templating here
```suggestion
    {{- .Values.config | tpl | nindent 4 }}
```

Comment (onedr0p) [reply charts/stable/frigate/templates/configmap.yaml]: Good point! Added.

File: charts/stable/frigate/values.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+ingress:
+  # -- Enable and configure ingress settings for the chart under this key.
+  # @default -- See values.yaml
+  main:
+    enabled: false
+
+securityContext:
+  # -- (bool) Privileged securityContext may be required if USB devices are accessed directly through the host machine
+  privileged: true
```
Comment (bjw-s) [inline charts/stable/frigate/values.yaml:46]:
```suggestion
  privileged:  # true
```
Missed this one earlier, in other charts we have the user explicitly set this:
https://github.com/k8s-at-home/charts/blob/09ab3726253bffb2b9bd7f6afc8bb129d0a4d285/charts/stable/zigbee2mqtt/values.yaml#L38

Comment (onedr0p) [reply charts/stable/frigate/values.yaml]: I just set it to false by default, that ok?

**Review by bjw-s — APPROVED** [review]

**Pull request merged**